debian/rules

```diff
@@ -1,13 +1,28 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-override_dh_auto_build:
-	md5sum -c md5sums.txt
+SOP_IMPL = sqop rsop gosop pgpainless-cli none
+SOP = $(firstword $(foreach cmd,$(SOP_IMPL),$(wildcard /usr/bin/$(cmd))))
+
+SIGNER_KEYRING = /usr/share/keyrings/debian-keyring.gpg
+ARCHIVE_CERTS = grml-archive-automatic-*.pgp
+ARCHIVE_CERTS_SIGS = $(wildcard keyrings/*.pgp.asc)
+ARCHIVE_KEYRING = grml-archive-keyring.pgp
+
+override_dh_auto_build: $(ARCHIVE_CERTS_SIGS)
+	@echo  $(ARCHIVE_CERTS_SIGS)
+	cat keyrings/$(ARCHIVE_CERTS) | \
+	  $(SOP) armor >keyrings/$(ARCHIVE_KEYRING)
 
 # package should work also on older releases like lenny, which
 # don't have support for xz yet
 override_dh_builddeb:
 	dh_builddeb -- -Zgzip
```

Comment:
Just checking while having you here :) - I suppose we could also drop this nowadays, right?

Reply:
For something targeting Debian, I think sure. For upstream I guess it will depend on what you want to support going back?

Reply:
Ah good point, thx. Hm, but I think any half decent Debian release has problems with that?

Reply:
I'm not sure I understand your last sentence. Do you mean have problems with xz or have problems with the override and thus gzip? Or something else?

Reply:
I'm invoking a magic ritual and reformulate the question as:

> What is the oldest Debian release we would support without the override, assuming we build the package on trixie or unstable?

Reply:
Ah! From deb(5), control.tar.xz support was added in dpkg 1.17.6 (Debian 8 jessie), and data.tar.xz support was added in dpkg 1.15.6 (Debian 6 squeeze). Since dpkg 1.19.0 (Debian 10 buster), dpkg-deb generates by default uniformly compressed .deb (so both control.tar and data.tar use the same compressor, before, only data.tar honored the compression option).